app/components/salesGPT.tsx

```diff
@@ -0,0 +1,78 @@
+"use client";
+
+require("../polyfill");
+
+import { useEffect, useState } from "react";
+import Sidebar from "../components/sidebar";
+import { isMobileScreen } from "../utils";
+import { useChatStore } from "../store";
+import styles from "../components/home.module.scss";
+import dynamic from "next/dynamic";
+import { Loading } from "./home";
+import { ErrorBoundary } from "./error";
+import { EmployeeItemProp } from "../function/Employees";
+
+const Settings = dynamic(async () => (await import("./settings")).Settings, {
+  loading: () => <Loading noLogo />,
+});
+
+const Employees = dynamic(async () => await import("./employees"), {
+  loading: () => <Loading noLogo />,
+});
+
+const useHasHydrated = () => {
+  const [hasHydrated, setHasHydrated] = useState<boolean>(false);
+
+  useEffect(() => {
+    setHasHydrated(true);
+  }, []);
+
+  return hasHydrated;
+};
+
+function _SalesGPT({ employees }: EmployeeItemProp) {
+  const config = useChatStore((state) => state.config);
+  useChatStore();
+
+  // Setting
+  const [openSettings, setOpenSettings] = useState(false);
+
+  const loading = !useHasHydrated();
+
+  if (loading) {
+    return <Loading />;
+  }
+
+  return (
+    <div
+      className={`${
+        config.tightBorder && !isMobileScreen()
+          ? styles["tight-container"]
+          : styles.container
+      }`}
+    >
+      <Sidebar title="SalgGpt" subTitle="" setOpenSettings={setOpenSettings}>
+        <></>
```

Comment:
Fjerne dette?

Reply:
Oi, for ett godt øye! Men vettuhva, uten linje 55 der så kan ikke yarn bygge, og vi får en feilmelding.
Jeg vet egentlig ikke hva som skjer annet enn at feilmeldingen er:
```
./app/components/salesGPT.tsx:54:8
Type error: Property 'children' is missing in type '{ title: string; subTitle: string; setOpenSettings: Dispatch<SetStateAction<boolean>>; }' but required in type '{ title: string; subTitle: string; setOpenSettings?: ((newState: boolean) => void) | undefined; createNewSession?: (() => void) | undefined; chatStore?: ChatStore | undefined; showSideBar?: boolean | undefined; setShowSideBar?: ((newState: boolean) => void) | undefined; children: ReactNode; }'.

  52 |       }`}
  53 |     >
> 54 |       <Sidebar
     |        ^
  55 |         title="SalgGpt"
  56 |         subTitle=""
  57 |         setOpenSettings={setOpenSettings}
error Command failed with exit code 1.
```
Og den er borte når <></> er tilstedet.